msgsecret.go

```diff
@@ -231,6 +231,23 @@ func (cli *Client) DecryptPollVote(ctx context.Context, vote *events.Message) (*
 	return &msg, nil
 }
 
+func (cli *Client) DecryptSecretEncryptedMessage(ctx context.Context, evt *events.Message) (*waE2E.Message, error) {
+	encMessage := evt.Message.GetSecretEncryptedMessage()
+	if encMessage == nil {
+		return nil, ErrNotSecretEncryptedMessage
+	}
+	plaintext, err := cli.decryptMsgSecret(ctx, evt, EncSecretEventEdit, encMessage, encMessage.GetTargetMessageKey())
```

Comment:
`EncSecretEventEdit` sounds too specific for `SecretEncryptedMessage`, should it actually detect the type or take it as a parameter? Or should the method be called `DecryptSecretEventEdit`?